run.sh

```diff
@@ -3,6 +3,7 @@
 set -eou pipefail
 
 TARGET_ENV="${TARGET_ENV:-development}"
+DEPLOY_KIND_OBSERVATORIUM="${DEPLOY_KIND_OBSERVATORIUM:-true}"
```

Comment:
What is this extra variable good for?

Reply:
On my machine, it takes time to deploy Kind .... so I set this variable to false and run again and again on the same kind and observatorium deployment without the need to redeploy:

```DEPLOY_KIND_OBSERVATORIUM=false make all```